controllers/users/authenticate-web.js

```diff
@@ -22,44 +22,67 @@ const authenticateWeb = async (req, res) => {
    * @type {RegisterBody}
    */
   const users = req.body;
-  users.username = 'auth_web_' + moment().format('YYYYMMDDHHmmssSSS');
+
+  const checkUser = await User.findOne({
+    where: {
+      human_id: users.human_id
+    }
+  });
 
   let insertedObject = {};
 
-  /**
-   * Inserting data to Postgres DB
-   */
-  try {
-    insertedObject = await sequelize.transaction(async (t) => {
-      const user = await UsersFunction.register(User, users, t);
-      const anonymousUser = await UsersFunction.registerAnonymous(User, user?.user_id, t);
+  if (checkUser) {
+    insertedObject = {
+      user: checkUser,
+      anonymousUser: await UsersFunction.findAnonymousUserId(User, checkUser.dataValues.user_id, {
+        raw: false
+      })
+    };
+  } else {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `findAnonymousUserId` function is called with `{raw: false}` as an option. However, in the summary of changes, it was mentioned that the default value for `options` is `{raw: true}`. If the function is not designed to handle `{raw: false}`, this could lead to unexpected behavior. Please ensure that the `findAnonymousUserId` function can handle different values for `raw`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->